README.md

```diff
@@ -32,27 +32,26 @@ Please see the [setup guide](SETUP.md) for more details on setting up your machi
 
 To setup on your local machine:
 
-1. Install Anaconda with Python >= 3.6. [Miniconda](https://conda.io/miniconda.html) is a quick way to get started.
-
-2. Clone the repository
+To install core utilities, CPU-based algorithms, and dependencies
```

Comment:
we should mention here which versions of python can be used, and ideally suggest using Anaconda or virtual env to provide isolation of dependencies, maybe point to some external documentation on how to set either of those up.

Reply:
Good points! It would be nice if we had a complete setup for the virtual env in a future PR but this is going to be a bit tricky.